browser/playlist/playlist_service_factory.cc

```diff
@@ -78,7 +79,9 @@ PlaylistServiceFactory::~PlaylistServiceFactory() = default;
 KeyedService* PlaylistServiceFactory::BuildServiceInstanceFor(
     content::BrowserContext* context) const {
   DCHECK(media_detector_component_manager_);
-  return new PlaylistService(context, media_detector_component_manager_.get());
+  return new PlaylistService(
+      context, HostContentSettingsMapFactory::GetForProfile(context),
```

Comment:
pls add `DependsOn(HostContentSettingsMapFactory::GetInstance());` to `PlaylistServiceFactory::PlaylistServiceFactory`

Reply:
Done.